ote_cli/ote_cli/datasets/__init__.py

```diff
@@ -1,35 +1,43 @@
+from typing import Optional, Type
+
+from ote_sdk.entities.datasets import DatasetEntity
+
 try:
+    ObjectDetectionDataset: Optional[Type[DatasetEntity]]
     from .object_detection.dataset import ObjectDetectionDataset
-except ImportError as e:
+except ImportError:
     ObjectDetectionDataset = None
 
 try:
+    ClassificationDataset: Optional[Type[DatasetEntity]]
     from torchreid.integration.sc.utils import ClassificationDatasetAdapter
+
     ClassificationDataset = ClassificationDatasetAdapter
-except ImportError as e:
+except ImportError:
     ClassificationDataset = None
 
 try:
+    SemanticSegmentationDataset: Optional[Type[DatasetEntity]]
     from .semantic_segmentation.dataset import SemanticSegmentationDataset
-except ImportError as e:
+except ImportError:
     SemanticSegmentationDataset = None
 
 assert any((ObjectDetectionDataset, ClassificationDataset, SemanticSegmentationDataset))
 
 
 def get_dataset_class(name):
     registry = {
-        'detection': ObjectDetectionDataset,
-        'classification': ClassificationDataset,
-        'segmentation': SemanticSegmentationDataset
+        "detection": ObjectDetectionDataset,
```

Comment:
This is because SDK is set to use double quotes, right?

Reply:
right